### PR TITLE
feat(nx-plugin): add executor to nx-plugin

### LINKED ISSUE #289: intro RC schema
this can be used in the nx-plugin to extend from

### PR DESCRIPTION
**Pull request by BioPhoton**
**Precondition:**
#740 
closes #289

**This PR includes:**
  - add executor `autorun`
  - add executor utils
  - add format normalization

**Not included in this PR:**
- e2e tests for the autorun executor
- [ ] #761

 
Related to:
#63

### CODE REVIEW

File: packages/models/src/lib/upload-config.unit.test.ts [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
 describe('uploadConfigSchema', () => {
   it('should accept a valid upload configuration', () => {
     expect(() =>
-      uploadConfigSchema.parse({
+      uploadConfigSchema().parse({
         apiKey: 'API-K3Y',
         organization: 'code-pushup',
         project: 'cli',
         server: 'https://cli-server.dev:3800/',
-      } satisfies UploadConfig),
```
Comment (BioPhoton) [inline packages/models/src/lib/upload-config.unit.test.ts:12]: It is hard to properly type the configurable schema, therefor i just removed the type here

Comment (BioPhoton) [reply packages/models/src/lib/upload-config.unit.test.ts]: reverted

File: project.json [hunk 2]
```diff
@@ -12,20 +12,14 @@
     },
     "code-pushup": {
       "command": "npx dist/packages/cli",
```
Comment (BioPhoton) [inline project.json:14]: This should already use the new executor ``, but our CI executes the released version. 
Therefor we have to change this in a followup PR.

File: knip.config.ts
Comment (matejchalk) [inline knip.config.ts:1]: Why did you remove this? Seems out of scope of this PR.

Also, if you're gonna remove Knip (no argument from me), then please also remove the NPM script and packages.

File: project.json [hunk 3]
```diff
@@ -12,20 +12,14 @@
...
-            "plugin-lighthouse",
-            "examples-plugins",
-            "react-todos-app"
-          ],
+      "options": {
+        "persist": {
+          "format": ["json"],
+          "filename": "code-pushup",
+          "outputDir": ".code-pushup",
           "target": "build"
```
Comment (matejchalk) [inline project.json:20]: This is in the wrong place, was part of `dependsOn`, here it means nothing.

```suggestion
```

File: project.json [hunk 4]
```diff
@@ -12,20 +12,14 @@
...
-            "plugin-js-packages",
-            "plugin-lighthouse",
-            "examples-plugins",
-            "react-todos-app"
-          ],
+      "options": {
+        "persist": {
+          "format": ["json"],
+          "filename": "code-pushup",
+          "outputDir": ".code-pushup",
```
Comment (matejchalk) [inline project.json:19]: I realize this is currently ignored because it doesn't run the executor, but why exclude `md` format and use `code-pushup` instead of the usual `report` as the filename? Please don't override the default values unless there's a good reason.

File: tools/scripts/zod-to-schema.ts [hunk 5]
```diff
@@ -0,0 +1,69 @@
+import { writeFile } from 'node:fs/promises';
+import { join } from 'node:path';
+import yargs from 'yargs';
+import { z } from 'zod';
+import { zodToJsonSchema } from 'zod-to-json-schema';
```
Comment (matejchalk) [inline tools/scripts/zod-to-schema.ts:5]: Should be installed with `npm i -D zod-to-json-schema`.

File: tools/scripts/zod-to-schema.ts [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+import { persistConfigSchema } from './packages/models/src';
```
Comment (matejchalk) [inline tools/scripts/zod-to-schema.ts:6]: Broken path:

```suggestion
import { persistConfigSchema } from '../../packages/models/src';
```

File: tools/scripts/zod-to-schema.ts [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    target: 'jsonSchema7',
+    definitions: {
+      persistOptions: persistConfigSchema,
+      //   uploadOptions: uploadConfigSchema,
+    },
+  });
+
+  const outputPath = join(outputDir, filename);
+  console.log(`Generated ${outputPath}`);
+  await writeFile(outputPath, JSON.stringify(jsonSchema));
```
Comment (matejchalk) [inline tools/scripts/zod-to-schema.ts:68]: Single-line `schema.json` is unreadable:

```suggestion
  await writeFile(outputPath, JSON.stringify(jsonSchema, null, 2));
```

File: tools/scripts/zod-to-schema.ts
Comment (matejchalk) [inline tools/scripts/zod-to-schema.ts:1]: Is this script actually usable? It generates something completely different to the executor schema and requires several fixed to run it succesfully. If it's a WIP, then please remove it from the PR, otherwise it just confuses things.

File: packages/nx-plugin/executors.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
+{
+  "executors": {
+    "autorun": {
+      "implementation": "./src/executors/autorun/executor",
+      "schema": "./src/executors/autorun/schema.json",
+      "description": "autorun executor"
```
Comment (matejchalk) [inline packages/nx-plugin/executors.json:6]: Could you add a more useful description? I already know I'm looking at an executor called autorun, so this description tells me nothing.

File: packages/nx-plugin/src/executors/autorun/Readme.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  }
+}
+```
+
+By default, the Nx plugin will derive the options from the executor config.
+
+The following things happen:
+
+- the format defaults to ['json']
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/Readme.md:24]: Again, why exclude `md`?

File: packages/nx-plugin/src/executors/autorun/Readme.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+- the output directory defaults to `${workspaceRoot}/.code-pushup/${projectName}`
+- the config file defaults to `${projectRoot}/code-pushup.config.ts`
+
+- // autorun options in project.json
+
+```json
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/Readme.md:30]: Use `jsonc` to get correct syntax highlighting for comments:

```suggestion
```jsonc
```

File: packages/nx-plugin/src/executors/autorun/Readme.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+
+## Usage
+
+// project.json
+
+```json
+{
+  "name": "my-project",
+  "targets": {
+    "autorun": {
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/Readme.md:13]: I think most users will want the target to be `code-pushup`, so that they know what they're running:

```suggestion
    "code-pushup": {
```

File: packages/nx-plugin/src/executors/autorun/executor.integration.test.ts [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+vi.mock('../internal/config', async () => {
+  const actual: any = await vi.importActual('../internal/config');
+
+  return {
+    ...actual,
+    persistConfig: vi.fn(actual.persistConfig),
+    uploadConfig: vi.fn(actual.uploadConfig),
+    globalConfig: vi.fn(actual.globalConfig),
+  };
+});
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/executor.integration.test.ts:29]: Just use spies.

File: packages/nx-plugin/src/executors/autorun/executor.integration.test.ts [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+  root: '.', // workspaceRoot
+  projectsConfigurations: {
+    projects: {
+      [projectName]: {
+        name: projectName,
+        root: `libs/${projectName}`,
+      },
+    },
+  },
+} as unknown as ExecutorContext;
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/executor.integration.test.ts:43]: Please stop using `as unknown as T`. I've complained about it so many times in your PRs, and I'm not going to stop. But I will stop refactoring it for you, it seems to be counterproductive :disappointed:

File: packages/nx-plugin/src/executors/autorun/executor.ts [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+  );
+  const command = createCliCommand(AUTORUN_COMMAND, cliArgumentObject);
+
+  const { dryRun } = options;
+  if (dryRun) {
+    logger.warn(`DryRun execution of: ${command}`);
+  } else {
+    try {
+      // eslint-disable-next-line n/no-sync
+      execSync(command, context.cwd ? { cwd: context.cwd } : {});
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/executor.ts:30]: Why not just use the async version? Is there some reason this needs to block the main thread?

File: packages/nx-plugin/src/executors/autorun/executor.integration.test.ts [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+        verbose: true,
+        persist: { filename: 'my-name' },
+        upload: {
+          server: 'https://new-portal.code-pushup.dev',
+        } as UploadConfig,
+      },
+      normalizedContext,
+    );
+  });
+});
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/executor.integration.test.ts:127]: Why is this tested here? The `parseAutorunExecutorOptions` function is defined in `utils.ts`.

File: packages/nx-plugin/src/executors/autorun/schema.json [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+    "projectPrefix": {
+      "type": "string",
+      "description": "Prefix for project name (default is ``)"
+    },
+    "persist": {
+      "type": "object",
+      "properties": {
+        "filename": {
+          "type": "string",
+          "description": "Filename to save the configuration to (default is `.codepushup.json`)"
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/schema.json:50]: Where did `.codepushup.json` come from?

File: packages/nx-plugin/src/executors/autorun/schema.json [hunk 17]
```diff
@@ -0,0 +1,100 @@
...
+        },
+        "outDir": {
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/schema.json:52]: Why give it a different name than in our config?

```suggestion
        "outputDir": {
```

File: packages/nx-plugin/src/executors/autorun/schema.json [hunk 18]
```diff
@@ -0,0 +1,100 @@
...
+          "type": "string",
+          "description": "Directory to save the configuration to (default is `.`)"
+        },
+        "format": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": ["json", "md"]
+          },
+          "description": "Format to save the configuration in (default is `json`)",
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/schema.json:62]: Saving the "configuration"? Surely you mean the "report".

File: packages/nx-plugin/src/executors/autorun/schema.json [hunk 19]
```diff
@@ -0,0 +1,100 @@
...
+    "persist": {
+      "type": "object",
+      "properties": {
+        "filename": {
+          "type": "string",
+          "description": "Filename to save the configuration to (default is `.codepushup.json`)"
+        },
+        "outDir": {
+          "type": "string",
+          "description": "Directory to save the configuration to (default is `.`)"
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/schema.json:54]: The default is `.`? Surely it's `.code-pushup`.

File: packages/nx-plugin/src/executors/autorun/schema.json [hunk 20]
```diff
@@ -0,0 +1,100 @@
...
+        },
+        "format": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": ["json", "md"]
+          },
+          "description": "Format to save the configuration in (default is `json`)",
+          "default": ["json"]
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/schema.json:63]: Again, why do you not want `report.md`s? :sweat_smile:

File: packages/nx-plugin/src/executors/autorun/utils.ts [hunk 21]
```diff
@@ -0,0 +1,30 @@
...
+} from './schema';
+
+export function parseAutorunExecutorOnlyOptions(
+  options: Partial<AutorunCommandExecutorOnlyOptions>,
+): AutorunCommandExecutorOnlyOptions {
+  const { projectPrefix, dryRun, onlyPlugins } = options;
+  return {
+    ...(projectPrefix == null ? {} : { projectPrefix }),
+    ...(dryRun == null ? {} : { dryRun: dryRun }),
+    ...(onlyPlugins ? { onlyPlugins } : {}),
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/utils.ts:15]:
```suggestion
    ...(projectPrefix && { projectPrefix }),
    ...(dryRun != null && { dryRun: dryRun }),
    ...(onlyPlugins && { onlyPlugins }),
```

File: packages/nx-plugin/src/executors/autorun/utils.unit.test.ts [hunk 22]
```diff
@@ -0,0 +1,94 @@
...
+  beforeAll(() => {
+    processEnvSpy = vi.spyOn(process, 'env', 'get').mockReturnValue({});
+  });
+
+  afterAll(() => {
+    processEnvSpy.mockRestore();
+  });
+
+  it('should leverage other config helper to assemble the executor config', () => {
+    processEnvSpy.mockReturnValue({ outputDir: 'from-env-vars' });
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/utils.unit.test.ts:57]: There's no assertion regarding this environment variable.

**Review by matejchalk — CHANGES_REQUESTED** [review]

File: tools/scripts/zod-to-schema.ts (hunk 5 above)
Comment (BioPhoton) [reply tools/scripts/zod-to-schema.ts:5]: I will delete this script

File: code-pushup.config.ts [hunk 23]
```diff
@@ -21,25 +21,7 @@ import {
...
 const config: CoreConfig = {
-  ...(env && {
-    upload: {
-      server: env.CP_SERVER,
-      apiKey: env.CP_API_KEY,
-      organization: env.CP_ORGANIZATION,
-      project: env.CP_PROJECT,
-    },
-  }),
-
```
[the remark this answers is not in the document]
Comment (BioPhoton) [reply code-pushup.config.ts:42]: it should be included in the executor logic. Unfo did not see it..

File: knip.config.ts
Comment (BioPhoton) [reply knip.config.ts:1]: hahaha... was just a mistake! But made me laugh.. ;)

File: project.json (hunk 4 above)
Comment (BioPhoton) [reply project.json]: was a debugging left over and i should have done a proper rescreening!

File: tools/scripts/zod-to-schema.ts (hunk 5 above)
Comment (BioPhoton) [reply tools/scripts/zod-to-schema.ts]: but for now reverted the changes

File: packages/nx-plugin/src/executors/autorun/schema.json (hunk 20 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/schema.json:63]: remofed the default and will pass nothing so CLI handles it

File: packages/nx-plugin/src/executors/autorun/schema.json (hunk 18 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/schema.json]: fixed

File: packages/nx-plugin/src/executors/autorun/schema.json (hunk 19 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/schema.json]: thx

File: packages/nx-plugin/src/executors/autorun/utils.ts (hunk 21 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/utils.ts:15]: Nice change! reads better and more compact

File: packages/nx-plugin/src/executors/autorun/utils.unit.test.ts (hunk 22 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/utils.unit.test.ts]: updated!

**Comment by BioPhoton** [comment]
@matejchalk related to the above mentioned failing action:

Had a bug with overwrites #753 and also think the action is maybe related?

The CI now throws for terminal argument parsing

File: tools/scripts/zod-to-schema.ts
Comment (BioPhoton) [reply tools/scripts/zod-to-schema.ts:1]: I forgot it. It was a WIP 🙈

File: packages/nx-plugin/executors.json (hunk 8 above)
Comment (BioPhoton) [reply packages/nx-plugin/executors.json]: Added better text

File: packages/nx-plugin/src/executors/autorun/Readme.md (hunk 9 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/Readme.md]: removed it

File: packages/nx-plugin/src/executors/autorun/Readme.md (hunk 11 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/Readme.md:16]: Thx! Was a "typo".

File: packages/nx-plugin/src/executors/autorun/executor.ts (hunk 14 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/executor.ts]: I wanted to keep that out of this PR as it includes proper forwarding of the logs and errors. Mentioned it in the PR description now.

File: packages/nx-plugin/src/executors/autorun/executor.integration.test.ts (hunk 15 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/executor.integration.test.ts]: moved!

File: packages/nx-plugin/src/executors/autorun/schema.json (hunk 17 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/schema.json]: OMG this was a mistake. In future a PR I will autogenerate the schema.json from types.

File: packages/nx-plugin/src/executors/autorun/schema.json (hunk 16 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/schema.json:50]: In the CLI but I guess I should just not mention defaults at all?

File: .github/workflows/code-pushup.yml [hunk 24]
```diff
@@ -36,4 +36,4 @@ jobs:
       - name: Run Code PushUp action
         uses: code-pushup/github-action@v0
         with:
-          bin: npx nx code-pushup --
+          bin: npx nx code-pushup
```
Comment (BioPhoton) [inline .github/workflows/code-pushup.yml:39]:
```suggestion
          bin: npx nx code-pushup --
```

File: packages/nx-plugin/src/executors/autorun/executor.integration.test.ts (hunk 12 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/executor.integration.test.ts]: done!

File: packages/nx-plugin/src/executors/autorun/executor.integration.test.ts (hunk 13 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/executor.integration.test.ts]: Sry for that! :(

File: packages/models/code-pushup.config.ts
Comment (matejchalk) [inline packages/models/code-pushup.config.ts:1]: I think this should be out of scope for this PR, per-project config is part of #706 .

File: packages/models/project.json [hunk 25]
```diff
@@ -46,6 +46,9 @@
       "cache": true,
       "inputs": ["production", "^production", "{projectRoot}/zod2md.config.ts"],
       "outputs": ["{projectRoot}/packages/models/docs/models-reference.md"]
+    },
+    "code-pushup": {
+      "executor": "@code-pushup/nx-plugin:autorun"
     }
```
Comment (matejchalk) [inline packages/models/project.json:52]: Same here, per-project targets are part of #706. I see you only added it for `models`, so I guess it's leftover from testing?

File: packages/nx-plugin/mock/utils/executor.ts [hunk 26]
```diff
@@ -0,0 +1,16 @@
+import { NormalizedExecutorContext } from '../../src/executors/internal/context';
+
+export function normalizedExecutorContext(
+  nameOrOpt: string | { projectName: string } = 'my-lib',
```
Comment (matejchalk) [inline packages/nx-plugin/mock/utils/executor.ts:4]: What's the sense in having a default value? I don't think that assuming there's a project called `my-lib` is likely to create a working configuration.

File: packages/nx-plugin/src/executors/autorun/Readme.md [hunk 27]
```diff
@@ -0,0 +1,60 @@
...
+- the CLI command is executed
+
+```jsonc
+{
+  "name": "my-project",
+  "targets": {
+    "code-pushup": {
+      "executor": "@code-pushup/nx-plugin:autorun",
+      "options": {
+        "projectPrefix": "my-project",
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/Readme.md:39]: We already know what the project `name` is, so how about making it a boolean flag? Then the name doesn't have to duplicated.

```suggestion
        "projectPrefix": true,
```

It's suggestion, though, and doesn't have to be in scope of this PR.

File: packages/nx-plugin/src/executors/autorun/Readme.md
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/Readme.md:1]: Please rename `Readme.md` to `README.md` for consistency.

File: packages/nx-plugin/src/executors/autorun/executor.ts [hunk 28]
```diff
@@ -0,0 +1,57 @@
...
+  terminalAndExecutorOptions: AutorunCommandExecutorOptions,
+  context: ExecutorContext,
+) {
+  const normalizedContext = normalizeContext(context);
+  const cliArgumentObject = parseAutorunExecutorOptions(
+    terminalAndExecutorOptions,
+    normalizedContext,
+  );
+  // console.log('AutorunCommandExecutorOptions: ', terminalAndExecutorOptions);
+  // console.log('cliArgumentObject: ', cliArgumentObject);
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/executor.ts:26]: Looks like leftover from debugging:

```suggestion
```

File: packages/nx-plugin/src/executors/autorun/utils.ts [hunk 29]
```diff
@@ -0,0 +1,30 @@
...
+  AutorunCommandExecutorOptions,
+} from './schema';
+
+export function parseAutorunExecutorOnlyOptions(
+  options: Partial<AutorunCommandExecutorOnlyOptions>,
+): AutorunCommandExecutorOnlyOptions {
+  const { projectPrefix, dryRun, onlyPlugins } = options;
+  return {
+    ...(projectPrefix && { projectPrefix }),
+    ...(dryRun != null && { dryRun: dryRun }),
```
Comment (matejchalk) [inline packages/nx-plugin/src/executors/autorun/utils.ts:14]:
```suggestion
    ...(dryRun != null && { dryRun }),
```

:top: I'm sure I've seen a lint rule for that, but can't remember what it was called :sweat_smile:

File: testing/test-utils/src/lib/utils/nx.ts [hunk 30]
```diff
@@ -0,0 +1,23 @@
+import { ExecutorContext } from '@nx/devkit';
+
+export function executorContext(
+  nameOrOpt: string | { projectName: string; cwd?: string } = 'my-lib',
+): ExecutorContext {
+  const { projectName, cwd = '' } =
```
Comment (matejchalk) [inline testing/test-utils/src/lib/utils/nx.ts:6]: Is an empty string a good default for `cwd`? I would expect `process.cwd()`, or leaving it undefined.

File: nx.json [hunk 31]
```diff
@@ -28,6 +28,12 @@
     "e2e": {
       "dependsOn": ["^build"]
     },
+    "code-pushup": {
+      "executor": "@code-pushup/nx-plugin:autorun",
+      "options": {
+        "projectPrefix": "cli"
+      }
+    },
```
Comment (matejchalk) [inline nx.json:36]: I don't think has any effect at present, so can probably be removed.

Tested with `npx nx code-pushup -- collect --onlyPlugins=js-packages` locally and saw from logs that only `collect` was being run, meaning the `autorun` executor is being ignored.

File: packages/models/project.json (hunk 25 above)
Comment (BioPhoton) [reply packages/models/project.json:52]: Actually I thought I can keep this one in as "test". Will remove it and add it to #706

File: packages/nx-plugin/mock/utils/executor.ts (hunk 26 above)
Comment (BioPhoton) [reply packages/nx-plugin/mock/utils/executor.ts:4]: Would also be more clear to read the test if I remove the default value.

File: packages/nx-plugin/src/executors/autorun/utils.ts (hunk 29 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/utils.ts:14]: One of this?

-  "prefer-object-spread/prefer-object-spread": "error",
- "object-shorthand": ["error", "always"],
- "no-useless-rename": "error"

File: testing/test-utils/src/lib/utils/nx.ts (hunk 30 above)
Comment (BioPhoton) [reply testing/test-utils/src/lib/utils/nx.ts]: fixed! left it undefined and adjusted the type

File: packages/nx-plugin/src/executors/autorun/utils.ts (hunk 29 above)
Comment (matejchalk) [reply packages/nx-plugin/src/executors/autorun/utils.ts]: Yes, that's what I was looking for :heart: It's even in the core set of rules, don't know how I missed it :sweat_smile: Created an issue to add those rules: https://github.com/code-pushup/eslint-config/issues/19

File: packages/nx-plugin/src/executors/autorun/Readme.md (hunk 27 above)
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/Readme.md]: Srythis was a typo. Makes no sense. I fixed the typo and added comments.

File: packages/nx-plugin/src/executors/autorun/Readme.md
Comment (BioPhoton) [reply packages/nx-plugin/src/executors/autorun/Readme.md:1]: Again.

File: packages/nx-plugin/mock/utils/executor.ts (hunk 26 above)
Comment (BioPhoton) [reply packages/nx-plugin/mock/utils/executor.ts]: removed default value

File: packages/models/project.json (hunk 25 above)
Comment (BioPhoton) [reply packages/models/project.json]: removed it
